<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile_view</class>
 <widget class="QWidget" name="profile_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>985</width>
    <height>594</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>985</width>
    <height>594</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit_profile">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
font: 700 15pt &quot;Microsoft JhengHei UI &quot;;
background-color: rgb(255, 170, 127);

</string>
   </property>
   <property name="text">
    <string>Edit Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_prompt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Microsoft JhengHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Microsoft JhengHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Infected?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Tel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_province">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_province">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_age">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nationality">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nationality">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Nationality</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>487</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_n_vaccinated">
      <property name="styleSheet">
       <string notr="true">
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
      <property name="text">
       <string>Non-Vaccinated</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_vaccinated">
      <property name="styleSheet">
       <string notr="true">
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
      <property name="text">
       <string>Vaccinated</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_infected">
      <property name="styleSheet">
       <string notr="true">
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
      <property name="text">
       <string>Infected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_death">
      <property name="styleSheet">
       <string notr="true">
font: 290 14pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
      <property name="text">
       <string>Death</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../OneDrive/เดสก์ท็อป/COJANG/user.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../OneDrive/เดสก์ท็อป/COJANG/user1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>290</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../OneDrive/เดสก์ท็อป/COJANG/doctor.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>-30</y>
     <width>311</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../OneDrive/เดสก์ท็อป/COJANG/syringe.png</pixmap>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label_name</zorder>
  <zorder>label_email</zorder>
  <zorder>pushButton_edit_profile</zorder>
  <zorder>label_status_prompt</zorder>
  <zorder>label_status</zorder>
  <zorder>label_tel</zorder>
  <zorder>label_province</zorder>
  <zorder>id</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>lineEdit_tel</zorder>
  <zorder>lineEdit_province</zorder>
  <zorder>lineEdit_id</zorder>
  <zorder>label_age</zorder>
  <zorder>lineEdit_age</zorder>
  <zorder>label_nationality</zorder>
  <zorder>lineEdit_nationality</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
